docs/reference/search-application/apis/get-search-application.asciidoc

```diff
@@ -8,7 +8,10 @@ beta::[]
 <titleabbrev>Get Search Application</titleabbrev>
 ++++
 
-Retrieves information about a Search Application.
+Retrieves information about a search application.
+
+If the search engine has an inconsistent state between its alias and configured indices, a warning header will be returned with the response.
```

Comment:
```suggestion
If the search application has an inconsistent state between its alias and configured indices, a warning header will be returned with the response.
```

Reply:
Thanks for the catch - I thought I was so good about not using engine anymore 😅